include/model/GPRModel.hpp

```diff
@@ -36,80 +36,38 @@
 #include "model/Model.hpp"
 #include "model/ModelFramework.hpp"
 #include "model/ScikitLearnFramework.hpp"
-#include "statespace/StateSpace.hpp"
 
 namespace libcozmo {
 namespace model {
 
+/// This class implements a Gaussian Process Regressor(GPR), a type of
+/// multi-dimensional linear regression model, from Scikit-Learn.
+///
+/// Given input vector action (a) : [speed, aspect ratio, edge offset], it
+/// predicts outcome delta state (Δs) : [delta distance, delta theta].
+/// such that the function can be defined as f: a--> Δs
 class GPRModel : public virtual Model {
```

Comment:
`This model learns the affect of and action a (i.e. the change in state). More specifically, it learns f: a -> delta s where a is _____ and delta s is ____.`